storage/cache/subtree_cache.go

```diff
@@ -25,6 +26,8 @@ import (
 	"github.com/google/trillian/storage/storagepb"
 )
 
+var populateConcurrency = flag.Int("populate_subtree_concurrency", 256, "Max number of concurrent workers concurrently populating subtrees")
```

Comment:
Can you TODO moving this flag up the stack or removing it after testing.

Reply:
done